src/Couchbase.Lite.Shared/Serialization/FleeceMutableDictionary.cs

```diff
@@ -80,17 +79,18 @@ public void Clear()
             }
 
             Mutate();
+
             _map.Clear();
-            foreach (var item in IterateDict()) {
-                _map[item.Key] = MValue.Empty;
-            }
 
-            Count = 0;
+            var list = IterateDict().AsParallel().ToList();
```

Comment:
`AsParallel()` ?  What is the goal of this?  The underlying logic is not parallel friendly